.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   build:
     if: github.repository == 'jazzband/django-revproxy'
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Probably should leave this change out here, and do it in #196.

Reply:
thank you for you comment @brianmay, I had to pin ubuntu 22.04 because ubuntu latest dropped python 3.7 support 😕

Reply:
Removed, thanks!